web-src/src/helpers/NetworkHelper.js

```diff
@@ -21,7 +31,7 @@ function unwrapError(error) {
 
 function wretchWithOptions(url) {
   return wretch(url)
-    .headers({ 'X-OW-EXTRA-LOGGING': 'on' })
```

Comment:
According to https://developer.adobe.com/app-builder/docs/guides/application_logging/#when-to-use-log-forwarding, the `X-OW-EXTRA-LOGGING` header should only be used during local development. Additionally, it impacts throttling limits and is primarily needed for debugging **successful** App Builder activations, as **failed** activations are logged by default. Therefore, I implemented a way to enable the storage of successful activations by setting the `appBuilderLogging` query search parameter. For more details, see the README

Reply:
Do you know if this parameter affects logs in the Dev Console? Having additional logs might be useful for us to troubleshoot potential issues.

Reply:
Are AppBuilder activations displayed somewhere in the Dev Console?